ProjectCoimbra.UWP/Project.Coimbra/Pages/GamePage.xaml.cs

```diff
@@ -214,5 +214,37 @@ private void Timer_Tick(object sender, object e)
                 }
             }
         }
+
+        /// Only pause the playback for SOLO mode
+        private async void Pause_Click(object sender, RoutedEventArgs e)
+        {
+            if (UserData.GameMode == UserData.Mode.Solo)
+            {
+                midiEngine.Pause();
+            }
+            ContentDialog dialog = new ContentDialog
+            {
+                Title = "Game paused",
```

Comment:
Add all static strings to the resources files so that they can be translated accordingly later. You can find examples of resource strings being used in the code